### PR TITLE
Add Rule to Detect Consecutive Newlines

### LINKED ISSUE #60: Detect consecutive blank lines
Similar to https://github.com/wikimedia/mediawiki-tools-codesniffer/commit/3a6709be2612fad63b9e9ead4e6644c28748edcc

### PR DESCRIPTION
**Pull request by mikeselander**
This PR copies in a rule to handles multiple empty lines all throughout PHP. I copied this rule in the MediaWiki ruleset as it felt unnecessary to include their whole ruleset for one very small rule and this is a particularly maintainable sniff. I am open to opinions on the approach.

This will detect any empty consecutive lines. It improves on the main Squiz rule as the Squiz rule only detects multiple lines within functions, and not in classes or in general PHP.

Fixes #60

### CODE REVIEW

File: HM/Sniffs/Whitespace/MultipleEmptyLinesSniff.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	public function register() {
+		return [ T_WHITESPACE ];
+	}
+
+	public function process( File $phpcsFile, $stackPtr ) {
+		$tokens = $phpcsFile->getTokens();
+
+		if ( $stackPtr > 2
+			&& $tokens[$stackPtr - 1]['line'] < $tokens[$stackPtr]['line']
+			&& $tokens[$stackPtr - 2]['line'] === $tokens[$stackPtr - 1]['line']
```
Comment (rmccue) [inline HM/Sniffs/Whitespace/MultipleEmptyLinesSniff.php]: Needs spacing inside `[]`; assuming this is the original formatting from MW?

Also, these conditionals should be flipped and broken up.

Comment (mikeselander) [reply HM/Sniffs/Whitespace/MultipleEmptyLinesSniff.php]: Ah ya, they were pulled over directly from the old rule. I'll clean them up a bit

Comment (mikeselander) [reply HM/Sniffs/Whitespace/MultipleEmptyLinesSniff.php]: What, exactly, would you like flipped and/or broken up in these conditionals @rmccue?

Comment (rmccue) [reply HM/Sniffs/Whitespace/MultipleEmptyLinesSniff.php]:
```
if ( $stackPtr > 2 && $tokens ... ) {
    do_the_thing();
```
➡️ 
```
if ( $stackPtr <= 2 ) {
    return;
}

if ( $tokens ... ) {
    return;
}
```

This should also reduce the line length significantly and mean you don't need to wrap them. (Of course, you can still combine the conditions, but often better to break them up for readability if they're this long.)

Comment (mikeselander) [reply HM/Sniffs/Whitespace/MultipleEmptyLinesSniff.php]: Gotcha, so early returning, that makes more sense. Thanks for clarifying!

**Pull request merged**